### ISSUE TITLE
Windows version outputs to wrong folder

### ISSUE THREAD

**Issue by steveduell** [cyanreg/cyanrip#11]
Hello.  I wonder if there is a problem with my command.  This is what I used:

cyanrip.exe -d D: -D "C:\Users\sduell\Desktop\Converted CD" -t 01 -o mp3 -b 256 -f -n

When the conversion is complete, I find "C:\Users\sduell\Desktop\Converted CD" is empty, but there is a new folder created called 'Converted CD [MP3]' instead, and the converted file is in there.  How can I make cyanrip output only into the folder I specify in the command, rather than into a newly generated one?

I'm using cyanrip version 0.1-rc2 on a Windows 10 box.

Thanks for your time.

**Comment by cyanreg** [comment]
Not really, the format is always appended at the end of the folder name. You can modify the source to remove that, but I don't see it making it into the master branch. Without this, ripping to multiple formats at once would be a mess with all files in the same directory.

**Comment by steveduell** [comment]
My issue wasn't with the format being appended to the directory name, just
that the directory I specify in the command always ends up empty.

I discovered the issue when I tried to convert a track and save it on the
desktop.  Running the command, CyanRip reported ripping and encoding the
track successfully, but when I went to look for the file on the desktop it
wasn't there.  It took me a long time to figure out that CyanRip went a
level UP and made a folder alongside the desktop called 'Desktop [MP3]'.

So my suggestion was that the multiple folders that CR creates should be
*inside* of the folder specified in the command, not outside it.  For
instance, if I make a folder called 'Converted CD' and specify it in the
command, then I would expect to find *'Converted CD [MP3]'*, '*Converted CD
[FLAC]*', '*Converted CD [OGG]*' folders INSIDE of that directory, not
outside.

Don't do it for my sake - I already wrote a python script that generates
the command for me and then cleans up the folders the way I want to see
it.  I was just providing feedback because I thought you might appreciate
it.

-Steve
